theme/Style/Style.qml

```diff
@@ -77,8 +80,8 @@ QtObject {
     readonly property int unifiedSearchResulIconLeftMargin: 12
     readonly property int unifiedSearchResulTitleFontSize: topLinePixelSize
     readonly property int unifiedSearchResulSublineFontSize: subLinePixelSize
-    readonly property string unifiedSearchResulTitleColor: "black"
-    readonly property string unifiedSearchResulSublineColor: "grey"
+    readonly property string unifiedSearchResulTitleColor: ncTextColor

```

Comment:
As far as I can see those `unifiedSearchResulTitleColor` and  `unifiedSearchResulSublineColor` are not used anymore and replaced with `ncTextColor` and `ncSecondaryTextColor` (or, mostly replaced), so, it might make sense to replace them all and remove them from the `Style.qml`?

Reply:
Good idea, done